packages/legend-extension-dsl-data-space/src/stores/query/DataSpaceQuerySetupStore.ts

```diff
@@ -155,8 +155,8 @@ export class DataSpaceQuerySetupStore extends QueryEditorStore {
     return true;
   }
 
-  override get isSaveActionDisabled(): boolean {
-    return true;
+  override get label(): string {
+    return 'DataSpace';
```

Comment:
I'm not sure if this label is useful or not for the user, maybe something else, I'm not sure what, but this definitely won't help.